### ISSUE TITLE
Create OONI glossary

### ISSUE THREAD

**Issue by agrabeli** [ooni/ooni.org#193]
OONI uses a lot of technical jargon, as well as terms which may create ambiguity (such as "middlebox").

To improve our communication, help resolve any ambiguity, and better support the work of translators, I think it would be great to have a glossary which explains all of the technical terms used by OONI. More specifically, I imagine this glossary containing a list of technical terms (used by OONI), and brief 1-2 sentence descriptions (in plain English). 

I plan to get started on this soon. Any suggestions/ideas are welcome!

**Comment by p1g30n** [comment]
I would start by curating a list of potentially ambiguous terms as has been started in https://github.com/TheTorProject/ooniprobe-android/issues/139 , let's maybe copy them here and add any missing candidates:

-     Middleboxes
-     Instant Messagging
-     NDT Speed Test
-     Performance

**Comment by p1g30n** [comment]
Suggested format for the glossary as to be universally readable by all ooni services and implementations  is **markdown** in the form of: 

```
### Some Term
Some definition for the term

### Some Other Term
Some other definition for the term
```

**Comment by agrabeli** [comment]
thanks @p1g30n :)

Additional terms we can include:

* Measurement
* Autonomous System Number (ASN)
* IP address
* Network anomaly
* Censorship
* HTTP
* DNS
* TCP
* Orchestration
* HTTP header
* DNS hijacking
* DNS spoofing
* DPI
* HTTP transparent proxy
* Proxy
* Tor hidden services
* Cloud-fronting
* HTTPS collectors
* Data processing pipeline
* Server
* Vantage point
* Test list
* Blocklist
* API
* Network measurement data
* Network interference
* Traffic manipulation
* Circumvention tools
* Metadata
* Raspberry Pi

I've included the above terms because community members have asked for their clarification on various occasions. I've also included the terms that our partners asked to be added to the glossary (during the OONI Partner Gathering): https://ooni.torproject.org/post/ooni-partner-gathering-2017/#sessions

We can use the following pad to add terms (and list them alphabetically): https://pad.riseup.net/p/ooni-glossary

**Comment by p1g30n** [comment]
Thanks for those @agrabeli. 
I assume a lot of terms are already (partly) explained in parts of the software, if that's the case it would be great to get pointed to those resources to reduce redundancy

**Comment by agrabeli** [comment]
The glossary has been written here: https://github.com/TheTorProject/ooni-web/pull/298

**Comment by agrabeli** [comment]
I have assigned @hellais for review.

**Comment by hellais** [comment]
This has been merged and deployed here: https://ooni.org/support/glossary. :whale:
